src/Format/Msg/Message.h

```diff
@@ -44,24 +44,12 @@ class Message
 
 public:
     Message();
+	Message(std::string &s, std::string &t);
```

Comment:
Why not `const std::string&`? Such interface make the intent of those arguments obscure. In all such cases either use `const std::string&` (read-only use or copy) or `std::string&&` (move). Here const reference is obvious choice.